homeassistant/components/meppel_afvalkalender/translations/bg.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Please remove all `translations/*.json` from your PR. This will be added via Lokalise. You can only keep the 'en.json' to make testing / development easier.

This will make reviewing a lot easier as well.

Reply:
done, thanks!

Reply:
About your ximmio.com integration comment: this was my initial feeling as well until I read that this was already 
proposed as a change in the original repo of Frenck, which he rejected because of all the area support you have to do.
https://github.com/frenck/python-twentemilieu/issues/4

On one hand I agree with him and additionally this makes it possible to have a more local branded integration (people from Meppel or Twente won't search for Ximmio and probably they even don't know about it). On the other hand having it all in one integration is less duplicate code, targets a bigger user group and more people from the community should take care of the changing area support together. So a requirement will be then to have min 1 developer per area that keeps it up-to-date for his/her area. 

Until it meets this requirement, I think separate integrations are the best solution.